{"uid":"898e89efe60124ae","name":"test_send_message","fullName":"test_Front.TestMessage#test_send_message","historyId":"969ef17ca6dfafaedf4eb3f9204b277d","time":{"start":1729961055840,"stop":1729961055840,"duration":0},"status":"broken","statusMessage":"TypeError: selenium.webdriver.remote.webdriver.WebDriver.find_element() argument after * must be an iterable, not WebElement","statusTrace":"request = <SubRequest 'init_driver' for <Function test_send_message>>\n\n    @pytest.fixture(scope=\"class\")\n    def init_driver(request):\n        # Инициализация драйвера\n        options = webdriver.ChromeOptions()\n        options.add_argument('--headless')\n        driver = webdriver.Chrome(options=options)\n    \n        driver.implicitly_wait(10)\n        driver.maximize_window()\n        request.cls.driver = driver\n    \n        # Инициализация страницы\n        login_page = LoginPage(driver)\n        home_page = HomePage(driver)\n    \n        # Открытие страницы\n        login_page.open_login_page()\n    \n        # Логинимся\n>       login_page.login_user()\n\nconftest.py:29: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nlogin_page.py:29: in login_user\n    self.send_keys(login_input, \"neonila.zinova@icloud.com\")\nbase_page.py:28: in send_keys\n    element = self.find_element(locator)\nbase_page.py:13: in find_element\n    return WebDriverWait(self.driver, time).until(EC.presence_of_element_located(locator),\n/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:96: in until\n    value = method(self._driver)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"3e9cbeb6c0128d4c9f87d254d0e6ef6a\")>\n\n    def _predicate(driver: WebDriverOrWebElement):\n>       return driver.find_element(*locator)\nE       TypeError: selenium.webdriver.remote.webdriver.WebDriver.find_element() argument after * must be an iterable, not WebElement\n\n/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py:84: TypeError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"init_driver","time":{"start":1729961055841,"stop":1729961059367,"duration":3526},"status":"broken","statusMessage":"TypeError: selenium.webdriver.remote.webdriver.WebDriver.find_element() argument after * must be an iterable, not WebElement\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/Dc/Dc/conftest.py\", line 29, in init_driver\n    login_page.login_user()\n  File \"/home/runner/work/Dc/Dc/login_page.py\", line 29, in login_user\n    self.send_keys(login_input, \"neonila.zinova@icloud.com\")\n  File \"/home/runner/work/Dc/Dc/base_page.py\", line 28, in send_keys\n    element = self.find_element(locator)\n  File \"/home/runner/work/Dc/Dc/base_page.py\", line 13, in find_element\n    return WebDriverWait(self.driver, time).until(EC.presence_of_element_located(locator),\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/selenium/webdriver/support/wait.py\", line 96, in until\n    value = method(self._driver)\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py\", line 84, in _predicate\n    return driver.find_element(*locator)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true}],"testStage":{"status":"broken","statusMessage":"TypeError: selenium.webdriver.remote.webdriver.WebDriver.find_element() argument after * must be an iterable, not WebElement","statusTrace":"request = <SubRequest 'init_driver' for <Function test_send_message>>\n\n    @pytest.fixture(scope=\"class\")\n    def init_driver(request):\n        # Инициализация драйвера\n        options = webdriver.ChromeOptions()\n        options.add_argument('--headless')\n        driver = webdriver.Chrome(options=options)\n    \n        driver.implicitly_wait(10)\n        driver.maximize_window()\n        request.cls.driver = driver\n    \n        # Инициализация страницы\n        login_page = LoginPage(driver)\n        home_page = HomePage(driver)\n    \n        # Открытие страницы\n        login_page.open_login_page()\n    \n        # Логинимся\n>       login_page.login_user()\n\nconftest.py:29: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nlogin_page.py:29: in login_user\n    self.send_keys(login_input, \"neonila.zinova@icloud.com\")\nbase_page.py:28: in send_keys\n    element = self.find_element(locator)\nbase_page.py:13: in find_element\n    return WebDriverWait(self.driver, time).until(EC.presence_of_element_located(locator),\n/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:96: in until\n    value = method(self._driver)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"3e9cbeb6c0128d4c9f87d254d0e6ef6a\")>\n\n    def _predicate(driver: WebDriverOrWebElement):\n>       return driver.find_element(*locator)\nE       TypeError: selenium.webdriver.remote.webdriver.WebDriver.find_element() argument after * must be an iterable, not WebElement\n\n/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py:84: TypeError","steps":[],"attachments":[{"uid":"e8b5fa10658e1328","name":"stdout","source":"e8b5fa10658e1328.txt","type":"text/plain","size":148}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":1,"hasContent":true},"afterStages":[],"labels":[{"name":"tag","value":"@pytest.mark.usefixtures('init_driver', 'base_url')"},{"name":"suite","value":"test_Front"},{"name":"subSuite","value":"TestMessage"},{"name":"host","value":"fv-az1019-33"},{"name":"thread","value":"2312-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test_Front"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["@pytest.mark.usefixtures('init_driver', 'base_url')"]},"source":"898e89efe60124ae.json","parameterValues":[]}